packages/server/graphql/mutations/helpers/notifications/notifyMattermost.ts

```diff
@@ -0,0 +1,309 @@
+      throw new Error('Invalid meeting type')
+  }
+}
+
+export const endMattermostMeeting = async (
+  meetingId: string,
+  teamId: string,
+  dataLoader: DataLoaderWorker
+) => {
+  const meeting = await dataLoader.get('newMeetings').load(meetingId)
+  const webhookUrl = await getWebhookForUserIdTeamId(meeting.facilitatorUserId, teamId, dataLoader)
+  if (!webhookUrl) return okResponse
+  const team = await dataLoader.get('teams').load(teamId)
+  const {facilitatorUserId: userId} = meeting
+  const summaryText = getSummaryText(meeting)
+  const meetingUrl = makeAppURL(appOrigin, `meet/${meetingId}`)
+  const attachments = [
+    makeFieldsAttachment(
+      [
+        {
+          short: true,
+          title: 'Team',
+          value: team.name
+        },
+        {
+          short: true,
+          title: 'Meeting',
+          value: meeting.name
+        },
+        {
+          short: false,
+          title: 'Summary',
+          value: summaryText
+        },
+        ...makeEndMeetingButtons(meeting)
+      ],
+      {
+        fallback: `Meeting completed, join: ${meetingUrl}`,
+        title: 'Meeting completed 🎉',
+        title_link: meetingUrl
+      }
+    )
+  ]
+  return await notifyMattermost('meetingEnd', webhookUrl, userId, teamId, attachments)
+}
+
+export const notifyMattermostTimeLimitStart = async (
+  scheduledEndTime: Date,
+  meetingId: string,
+  teamId: string,
+  dataLoader: DataLoaderWorker
+) => {
+  const meeting = await dataLoader.get('newMeetings').load(meetingId)
+  const webhookUrl = await getWebhookForUserIdTeamId(meeting.facilitatorUserId, teamId, dataLoader)
+  if (!webhookUrl) return okResponse
+
+  const team = await dataLoader.get('teams').load(teamId)
+  const {name: meetingName, phases, facilitatorStageId, facilitatorUserId: userId} = meeting
+  const {name: teamName} = team
+  const stageRes = findStageById(phases, facilitatorStageId)
+  const {stage} = stageRes!
+  const meetingUrl = makeAppURL(appOrigin, `meet/${meetingId}`)
+  const {phaseType} = stage
+  const phaseLabel = phaseLabelLookup[phaseType]
+
+  const fallbackDate = formatWeekday(scheduledEndTime)
+  const fallbackTime = formatTime(scheduledEndTime)
+  const fallbackZone = Intl.DateTimeFormat().resolvedOptions().timeZone || 'Eastern Time'
+  const fallback = `${fallbackDate} at ${fallbackTime} (${fallbackZone})`
+  const constraint = `You have until *<!date^${toEpochSeconds(
+    scheduledEndTime
+  )}^{date_short_pretty} at {time}|${fallback}>* to complete it.`
+
+  const attachments = [
+    makeFieldsAttachment(
+      [
+        {
+          short: true,
+          title: 'Team',
+          value: teamName
+        },
+        {
+          short: true,
+          title: 'Meeting',
+          value: meetingName
+        },
+        {
+          short: false,
+          value: constraint
+        },
+        {
+          short: false,
+          title: 'Link',
+          value: `[https:/prbl.in/${meetingId}](${meetingUrl})`
```

Comment:
`prbl.in` only works for our cloud offering, not PPMI or staging servers.
might i suggest `process.env.INVITATION_SHORTLINK`

Reply:
still a bug!